apps/mobile/src/app/(home)/settings/index.tsx

```diff
@@ -64,10 +65,14 @@ export default function SettingsScreen() {
             id: 'settings.display.cell_title',
             message: 'Display',
           })}
-          caption={t({
-            id: 'settings.display.cell_caption',
-            message: 'Theme, bitcoin unit and more',
-          })}
+          caption={
```

Comment:
I added this handling of text as it says `Theme, bitcoin unit and more` but that's not correct if we hide the other settings